lib/booker/client.rb

```diff
@@ -321,6 +321,13 @@ def results_from_response(response, booker_model=nil)
         parsed_response
       end
 
+      def response_is_error?(response, http_method)
+        successful_delete_request = (http_method == :delete) && (response.code == 204)
+        return false if successful_delete_request
```

Comment:
this could be just `return false if (http_method == :delete) && (response.code == 204)` and we don't need the otherwise unused variable.

Reply:
I think it makes it more readable

Reply:
I disagree, it just clutters up the short circuit into two lines. 

If you make successful_delete_request into a method `successful_delete_request?` and put the `(http_method == :delete) && (response.code == 204)` in there, *then* it would be more readable. As it stands, you're making me read 2 lines of code where one would do.